lms/djangoapps/discussion/static/discussion/js/discussion_router.js

```diff
@@ -50,7 +50,7 @@
 
                     Backbone.history.start({
                         pushState: true,
-                        root: '/courses/' + this.courseId + '/discussion/forum/'
+                        root: this.rootUrl
```

Comment:
As mentioned before, this is how it should be. The only place that uses Python conventions is in the factory which is rendered by Mako.